gulp/config.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
       loc.src + '/js/**/*.js'
     ],
     dest: loc.dist + '/js/',
-    name: 'main.js'
+    name: 'crtool.js'
```

Comment:
I suggest renaming this file to make it clear that it only applies to the start page. Maybe `crt-start.js`?

Reply:
changed to crt-start.js